models/project_repository.go

```diff
@@ -0,0 +1,188 @@
+package models
+
+import (
+	"log"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/project"
+	"github.com/jinzhu/gorm"
+	satoriuuid "github.com/satori/go.uuid"
```

Comment:
@aslakknutsen I've used this import name because VSCode at some point imported the goa uuid package which confused me for a long time.